arangod/Aql/Query.cpp

```diff
@@ -1466,16 +1466,17 @@ graph::Graph const* Query::lookupGraphByName(std::string const& name) {
   }
   graph::GraphManager graphManager{_vocbase, _contextOwnedByExterior};
 
-  auto g = graphManager.lookupGraphByName(name);
+  auto result = graphManager.lookupGraphByName(name);
 
-  if (g.fail()) {
+  if (result.fail()) {
     return nullptr;
   }
 
-  auto graph = g.get().get();
-  _graphs.emplace(name, std::move(g.get()));
+  auto uniuqe_graph = std::move(result.get());
```

Comment:
The solution now is more complicated than before, so why is it necessary.
Apart from that:
* `uniuqe` -> `unique`
* please follow our variable naming conventions